Exit code: 1
Stdout:

Search operators to employ:
local_random_walk(scale=1.0,distribution='uniform',beta=1.5) with probabilistic
swarm_dynamic(factor=0.7,self_conf=2.54,swarm_conf=2.56,version='inertial',distribution='uniform') with greedy
--------------------------------------------------

Stderr:
Traceback (most recent call last):
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/ollama_output_20241029_090042/execution_iteration_3.py", line 35, in <module>
    evaluate_sequence_IOH(heur, problem_id, instance, dimension, num_agents, num_iterations, num_replicas)
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/ollama_output_20241029_090042/execution_iteration_3.py", line 19, in evaluate_sequence_IOH
    met.run()
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/metaheuristic.py", line 143, in run
    self.apply_search_operator(perturbator, selector)
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/metaheuristic.py", line 99, in apply_search_operator
    exec('Operators.' + operator_name + '(self.pop,' + operator_params)
  File "<string>", line 1, in <module>
TypeError: local_random_walk() got an unexpected keyword argument 'beta'
